src/libs/StringUtils.ts

```diff
@@ -74,6 +74,22 @@ function normalizeAccents(text: string) {
     return text.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
 }
 
+/**
+ * Normalize a string by:
+ * - removing diacritical marks
+ * - Removing non-alphabetic and non-latin characters from a string
+ * - Removing invisible characters
+ * - normalizing space-like characters into normal spaces
+ * - collapsing whitespaces
+ * - trimming
+ */
+function normalize(text: string): string {
```

Comment:
Can this reuse some of the other normalize functions? Like for diacritics?

Reply:
It does reuse `sanitizeString` and `removeInvisibleCharacters`. 

I agree there's kind of a lot of overlap here that should probably be cleaned up at some point. Don't really want to do it in the scope of a deploy blocker fix though. I feel like long-term we'll want a really fast fuzzy matcher that handles normalization internally.